SummerProject/article/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from user.models import User
+from category_team.models import Team, Category, SocialNetworkChoice
```

Comment:
Please rename this to `categories` instead of `category_team`

Reply:
Done